### PR TITLE
Update Zulip

### LINKED ISSUE #3604: Update: Zulip icon
**Name:** Zulip
**Official resources for icon and color:** https://github.com/zulip/zulip/blob/master/static/images/zulip-logo.svg

### PR DESCRIPTION
**Pull request by fbernhart**
zulip

Closes #3604 
**Alexa rank:** [318,963](https://www.alexa.com/siteinfo/zulip.com)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
The old icon was outdated, as they have a new icon now.

**Source:** https://github.com/zulip/zulip/tree/master/static/images/logo
**Color:** I've used white `#FFFFFF`, as it's the color of the letter Z in the SVG file as well as the color used in the logo in the [website header](https://zulip.com/). The website also uses all kinds of shades of green., so I'm open for discussion.

### CODE REVIEW

**Comment by adamrusted** [comment]
Their [readthedocs](https://zulip.readthedocs.io/) uses `#2980B9`, which features on the website also. @simple-icons/maintainers - any other ideas?

**Comment by mondeja** [comment]
`#2980B9` it's the default color of ReadtheDocs's default theme. See here for example: https://sphinx-rtd-theme.readthedocs.io/en/latest/contributing.html

**Comment by adamrusted** [comment]
> `#2980B9` it's the default color of ReadtheDocs' default theme.

My bad - are we good with `#ffffff` as the color then?

**Comment by mondeja** [comment]
> are we good with `#ffffff` as the color then?

I think so, since none of the green seems to be the main :thinking:

File: _data/simple-icons.json [hunk 1]
```diff
@@ -7327,8 +7327,8 @@
         },
         {
             "title": "Zulip",
-            "hex": "52C2AF",
-            "source": "https://github.com/zulip/zulip/"
+            "hex": "FFFFFF",
+            "source": "https://github.com/zulip/zulip/tree/master/static/images/logo"
```
Comment (adamrusted) [inline _data/simple-icons.json:7331]: Is it worth updating the source to the website? They now include the SVG logo on the homepage.

File: icons/zulip.svg [hunk 2]
```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Zulip icon</title><path d="M24 12a12 12 0 1 1-24 0 12 12 0 0 1 24 0zm-8.03-6H7.02l1 2.03h6.06l-7.06 7.5 1 2.46h8.96l-1-2.02H9.91l7.06-7.5-1-2.46z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Zulip icon</title><path d="M22.7662 20.4104c0 1.9744-1.4484 3.5896-3.2184 3.5896H4.4522c-1.77 0-3.2184-1.6152-3.2184-3.5896 0-1.2087.5434-2.2835 1.3707-2.9345l8.0323-7.1733c.1494-.1279.3431.0783.2355.2505L7.926 16.4525c-.0825.165.0245.3682.1943.3682h11.4282c1.77 0 3.2178 1.6158 3.2178 3.5897zm0-16.8208c0 1.2087-.5434 2.2835-1.3707 2.9345l-8.0323 7.1733c-.1494.1279-.3431-.0783-.2355-.2505l2.9464-5.8994c.0825-.165-.0245-.3682-.1943-.3682H4.4522c-1.77 0-3.2184-1.6152-3.2184-3.5897C1.2338 1.6152 2.6822 0 4.4522 0h15.0962c1.77 0 3.2178 1.6152 3.2178 3.5896z"/></svg>
```
Comment (adamrusted) [inline icons/zulip.svg:1]: This is to a precision of 4 - any chance you could reduce it down?

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (ericcornelissen) [reply _data/simple-icons.json:7331]: I think using GitHub as a source is better, it'll probably be easier to find an updated version of the logo there if we need it.

However, as per [the Contributing Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines), this URL should include the commit hash of the commit in which the SVG that we used was added.

**Comment by fbernhart** [comment]
Done. I've changed the source to include the commit hash as well as reduced it down to a precision of 3. 😊

**Review by adamrusted — APPROVED** [review]
LGTM! 💤

**Pull request merged**